core/src/com/unciv/logic/civilization/PolicyManager.kt

```diff
@@ -225,4 +229,21 @@ class PolicyManager {
         }
         return freeBuildings
     }
+
+    private fun triggerGlobalAlerts(policy: Policy) {
+        var extraNotificationText = ""
+        // Not only is this an anti-pattern, it is the wrong anti-pattern :D
+        if (policy.name == "Patronage Branch Complete") {
+            extraNotificationText = "\nThis causes our influence with City-States to drop faster!"
+            // Preferably this should go in the JSON file, but I don't feel at ease editing such basic data types as the Policy class yet.
+        }
+        for (civ in civInfo.gameInfo.civilizations) {
+                if (civ == civInfo) continue
+            if (civ.getKnownCivs().contains(civInfo)) {
+                civ.addNotification("${civInfo.civName} has adopted the ${policy.name} policy$extraNotificationText", NotificationIcon.Culture)
```

Comment:
This is tricky. You're adding two strings together for the notification, you can't .tr() the two strings before but also you need to ensure that the final text is translatable.
I would probably go with wrapping both texts in {}, like b = "{b...}", finalText= "{a...}$b"
Also I would go with var notificationText = if() "..." else "..."; civ.addNotification("$notificationText}$extraNotificationText")